### DISCUSSION TITLE
Installing Big Sur guest on Ventura host

### DISCUSSION THREAD

**Discussion by sjmadsen** [utmapp/UTM#5293] [UTM Help (macOS)]
I'm trying to install a Big Sur guest on UTM 4.2.5 on an M1 running Ventura.

Choosing the virutalization option and an `.ipsw` restore image for Big Sur 11.6 results in "your machine does not support running this IPSW". I believe the image I chose was for the first M1 Mac Mini, if that matters.

Choosing emulation and using an appropriate Big Sur `.iso`, I can't get past the UEFI boot screen. Exiting the shell and going through the EFI menus to boot from a file doesn't work because it doesn't find any files or devices at all.

Am I doing something wrong or is this no longer possible?

**Comment by tbussmann** [comment]
AFAIK there is no way to virtualise Big Sur on aarch64 (and there never has been).
